cal-itp/eligibility-server
# Move Docker files into .devcontainer

closes #91 

## What this PR does
- Moves Docker and related files into .devcontainer, and updates files accordingly
- Updates docs with instruction to cd into .devcontainer

## How to test
- Run `cd .devcontainer` and then `docker compose --no-cache build server` and then `docker compose --no-cache build dev`
- Reopen the repository in VS Code Remote Container. Ensure that pre-commit is installed and running.
- Run the app in Run and Debug mode
- Run `python setup.py` - Should run successfully
- Run `coverage run -m pytest -m databasetest` - All tests should pass
- Run `source tests/.env.settingstest` and then `coverage run -m pytest -m settingstest` - All tests should pass
- Test that editing files locally works
